packages/eslint-plugin-dialtone/lib/rules/recommend-typography-style.js

```diff
@@ -32,7 +32,20 @@ module.exports = {
       VAttribute(node) {
         if (node.key.name === 'class') {
           const classes = node.value.value.split(' ');
-          const typographyClasses = ['d-fs', 'd-fw', 'd-lh', 'd-ff'];
+          const typographyClasses = [
+            'd-fs',
+            'd-fw-normal',
+            'd-fw-medium',
+            'd-fw-semibold',
+            'd-fw-bold',
+            'd-fw-unset',
```

Comment:
`d-fw-unset` is actually a flex wrap utility class https://dialtone.dialpad.com/utilities/flex/direction-wrap-flow.html#flex-wrap

```suggestion
```

Reply:
Actually just found that we have it duplicated.

![image](https://github.com/user-attachments/assets/e313ae68-a1e3-4b73-bf93-a54a1d6be484)
![Screenshot 2024-09-10 at 6 24 29 p m](https://github.com/user-attachments/assets/57b96c5f-091b-4f1e-9c1d-b45c34d9284f)

The thing is we don't have font weight d-fw-unset documented https://dialtone.dialpad.com/utilities/typography/font-weight.html#classes

Reply:
Oh, I didn't realize this, thanks for catching it!